deploy/k8s-onprem/templates/deployment.yaml

```diff
@@ -79,12 +79,25 @@ spec:
             - containerPort: 8002
               name: metrics
           livenessProbe:
+            initialDelaySeconds: 15
+            failureThreshold: 3
+            periodSeconds: 10
```

Comment:
These are already the defaults. Adding them verbose, so that people could customize the values easier.